gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-api/src/main/snapshot/org.apache.gobblin.service.flowstatuses.snapshot.json

```diff
@@ -38,14 +38,14 @@
     "doc" : "Execution status for a flow or job",
     "symbols" : [ "COMPILED", "PENDING", "PENDING_RETRY", "ORCHESTRATED", "RUNNING", "COMPLETE", "FAILED", "CANCELLED" ],
     "symbolDocs" : {
-      "COMPILED":"Flow compiled to jobs.",
-      "PENDING":"Flow or job is in pending state.",
-      "PENDING_RETRY":"Flow or job is pending retry.",
-      "ORCHESTRATED":"Job(s) orchestrated to spec executors.",
-      "RUNNING": "Flow or job is currently executing",
-      "COMPLETE":"Flow or job completed execution",
-      "FAILED":"Flow or job failed",
-      "CANCELLED":"Flow cancelled."
+      "COMPILED" : "Flow compiled to jobs.",
```

Comment:
Are the whitespaces intended?

Reply:
Yeah, it generates the files with those spaces so makes it consistent with the other files.